### ISSUE TITLE
Pretrained MSDS-RCNN model

### ISSUE THREAD

**Issue by anushl9o5** [Li-Chengyang/MSDS-RCNN#4]
Hi, I would like to know if it is possible for you to share the **pre-trained model of the MSDS-RCNN**. I realize you have shared a basic VGG model trained on both annotations.
But I am looking to get the reported MR (11% and 7%) according to the paper. If not possible, please share **only the proposals** of the MSDS-RCNN
Thanks!.

**Comment by Li-Chengyang** [comment]
Actually we did share the pre-trained model of MSDS-RCNN. We use the word 'VGG16 model' just mean that our model was built upon a VGG16 basemodel.

**Comment by anushl9o5** [comment]
@Li-Chengyang 
Oh okay, my mistake sorry. 
But I am still confused because I'm not able to get the reported MR in the paper. 
Could you yell me what test set and GT was used to get 11%?
Thanks!

**Comment by Li-Chengyang** [comment]
11% MR was obtained on the original test set but we used improved test annotations published by Jingjing Liu. You can find a download link in our paper or you can directly download from Liu's website.

**Comment by anushl9o5** [comment]
@Li-Chengyang oh okay. 
So the improved vgg model has been trained on the same improved annotations? But only the improved test set annotations are available. Will the improved training annotations be made available? 

Also is it possible to share the proposals of the MSDS-RCNN?

**Comment by Li-Chengyang** [comment]
In our project page, we have provided download links of all used data. There is no private data.
For proposals, as I can recall, we get ~99% recall using 50 proposals per image. I didn't save proposal files. Now that we have provided pretrained models, maybe you can generate proposal files yourself.
